src/Idris/IBC.hs

```diff
@@ -735,8 +732,11 @@ processAccess :: Bool -- ^ Reexporting?
            -> IBCPhase
            -> Archive -> Idris ()
 processAccess reexp phase ar = do
+    logIBC 3 $ "processAccess (reexp, phase)" ++ show (reexp, phase)
     ds <- getEntry [] "ibc_access" ar
+    logIBC 3 $ "processAccess ds" ++ show ds
     mapM_ (\ (n, a_in) -> do
```

Comment:
I know it is not you that introduced this, but would it be possible to use a `where`-clause to define the huge anonymous function in the lambda instead?

Reply:
I thought about that myself but there are many other functions in `Idris.IBC` using precisely the same form.

Reply:
I do not think it is blocking to fix this, but it may be useful if you have time.
Of course it would be nice to clean up the other functions as well, but maybe you could kindly just create an issue for those?

Reply:
I hope not otherwise the original code should have been rejected ;).

Happy to raise an issue. Was thinking it might be nicer even to use `forM_` and not having to introduce a local definition via where.

Reply:
You are welcome to raise the issue.

Reply:
Might be better to work on a PR to refactor those functions. I'd be happy to put that together once this PR goes through. Less bureaucracy that way and I might get unlucky with merge conflicts if someone eagerly picks up the issue.